lib/byron/test/data/cardano-node/node.config

```diff
@@ -0,0 +1,73 @@
+# vim: set filetype=yaml:
+# -*- mode: yaml -*-
+
+#     _   _           _          ____             __ _
+#    | \ | | ___   __| | ___    / ___|___  _ __  / _(_) __ _
+#    |  \| |/ _ \ / _` |/ _ \  | |   / _ \| '_ \| |_| |/ _` |
+#    | |\  | (_) | (_| |  __/  | |__| (_) | | | |  _| | (_| |
+#    |_| \_|\___/ \__,_|\___|   \____\___/|_| |_|_| |_|\__, |
+#                                                      |___/
+
+NodeId:
+NumCoreNodes: 1
+PBftSignatureThreshold: 1000000000000000
+Protocol: RealPBFT
+RequiresNetworkMagic: RequiresMagic
+TurnOnLogMetrics: False
+TurnOnLogging: True
+ViewMode: SimpleView
+
+
+#     _   _           _       _          ____
+#    | | | |_ __   __| | __ _| |_ ___   |  _ \ __ _ _ __ __ _ _ __ ___  ___
+#    | | | | '_ \ / _` |/ _` | __/ _ \  | |_) / _` | '__/ _` | '_ ` _ \/ __|
+#    | |_| | |_) | (_| | (_| | ||  __/  |  __/ (_| | | | (_| | | | | | \__ \
+#     \___/| .__/ \__,_|\__,_|\__\___|  |_|   \__,_|_|  \__,_|_| |_| |_|___/
+#          |_|
+
+ApplicationName: cardano-sl
+ApplicationVersion: 1
+LastKnownBlockVersion-Major: 0
+LastKnownBlockVersion-Minor: 2
+LastKnownBlockVersion-Alt: 0
+
+
+#     _                      _
+#    | |    ___   __ _  __ _(_)_ __   __ _
+#    | |   / _ \ / _` |/ _` | | '_ \ / _` |
+#    | |__| (_) | (_| | (_| | | | | | (_| |
+#    |_____\___/ \__, |\__, |_|_| |_|\__, |
+#                |___/ |___/         |___/
+
+# global filter; messages must have at least this severity to pass:
+minSeverity: Info
+
+# if not indicated otherwise, then messages are passed to these backends:
+defaultBackends:
+  - KatipBK
+
+# if not indicated otherwise, then log output is directed to this:
+defaultScribes:
+  - - StdoutSK
+    - stdout
+
+# more options which can be passed as key-value pairs:
+options:
+  # Disable "Critical" logs that are actually metrics...
+  mapBackends:
+    cardano.node-metrics: []
+    cardano.node.BlockFetchDecision.peers: []
+    cardano.node.ChainDB.metrics: []
+    cardano.node.metrics.ChainDB: []
+    cardano.node.metrics: []
+    cardano.node.metrics: []
+
+# these backends are initialized:
+setupBackends:
+  - KatipBK
+
+# here we set up outputs of logging in 'katip':
+setupScribes:
+  - scName: stdout
+    scKind: StdoutSK
+    scFormat: ScText
```

Comment:
still touched by the not-so-user-friendly level of sophistication needed to set logging....

Reply:
:no_mouth: